package.json

```diff
@@ -83,7 +83,7 @@
     "jest-enzyme": "^6.0.2",
     "license-check-and-add": "^2.3.6",
     "markdownlint-cli": "^0.13.0",
-    "prettier": "1.14.2",
+    "prettier": "1.14.3",
```

Comment:
why do we have it pinned?

Reply:
that's an excellent question - let me fix that

Reply:
@nadiia should be fixed